ledger/participant-integration-api/src/main/scala/platform/index/BuffersUpdater.scala

```diff
@@ -48,7 +47,7 @@ private[index] class BuffersUpdater(
   private val logger = ContextualizedLogger.get(getClass)
 
   private[index] val updaterIndex = new AtomicReference(
-    Offset.beforeBegin -> EventSequentialId.beforeBegin
+    Offset.beforeBegin -> 0L // TODO: append-only: FIXME consolidating parameters table
```

Comment:
The file where this was defined is removed. I would prefer the keep the removal of that file, since this PR is mostly moving around stuff, and if that is okay with you I would prefer to not introduce any extra point of reuse ATM, since that belongs to the task about consolidating the parameters_table related part.
FYI: just made a quick search in the code, and there are quite some places where we use 0L directly instead of the previously introduced `EventSequentialId` (besides what I introduced in this PR with the TODOs), so as doing the consolidation it would be also nice to bring them all under the same umbrella.